linode/manager
# chore: [M3-6024] – Upgrade NR to v1230

## Description 📝
Although the error described in the ticket does not seem to be happening anymore, I did notice that we were significantly behind on the NR version we were using (v1184).

This PR brings us up to the latest NR version (v1230).

## How to test 🧪
I believe the easiest way to test this locally would be to:

1. Set this condition to `true`:
https://github.com/linode/manager/blob/8be3dc4fae584c3d16f2880716717eaf9b8edb4a/packages/manager/src/App.tsx#L82

2. Go to your `localhost` and check the network requests. You should see POST requests to `bam.nr-data.net` and somewhere in the URL you should see `v=1.230.0` (this confirms the version upgrade)

(note: depending on your browser configurations, these requests may be filtered out and won't display. For example, I don't see the requests in Chrome, but I see them in Firefox)
